src/frontend/screens/Settings/components/ClearCache.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { InfoBox } from 'frontend/components/UI'
+import ContextProvider from 'frontend/state/ContextProvider'
+import { CleaningServicesOutlined } from '@mui/icons-material'
+
+const ClearCache = () => {
+  const { refreshLibrary } = useContext(ContextProvider)
+  const { t } = useTranslation()
+
+  async function clearHeroicCache() {
+    const storage: Storage = window.localStorage
+    storage.removeItem('updates')
+    window.api.clearCache(true)
+    return refreshLibrary({ runInBackground: true })
+  }
+
+  return (
+    <>
+      <h3 className="settingSubheader">
+        {t('settings.advanced.title.clearCache', 'Clear Cache')}
+      </h3>
+      <InfoBox text={t('settings.advanced.details', 'Details')}>
+        {t(
+          'settings.advanced.clearCache.help1',
+          'This action will clear the following caches:'
+        )}
+        <ul>
+          <li>
+            {t(
+              'settings.advanced.clearCache.help2',
+              'Third-party game info (scores, steam compatibility, howlongtobeat, pcgamingwiki, applegamingwiki)'
+            )}
+          </li>
+          <li>
+            {t(
+              'settings.advanced.clearCache.help3',
+              'Legendary library info (list of games, install dialog info, game info)'
+            )}
+          </li>
+          <li>
+            {t(
+              'settings.advanced.clearCache.help4',
+              'GOG library info (list of games, install dialog info, api info -i.e: requirements-)'
+            )}
+          </li>
+          <li>
+            {t(
+              'settings.advanced.clearCache.help5',
+              'Amazon library info (list of games, install dialog info)'
+            )}
+          </li>
+        </ul>
+        {t('settings.advanced.clearCache.help6', 'This will NOT delete:')}
+        <ul>
+          <li>{t('settings.advanced.clearCache.help7', 'Store login')}</li>
+          <li>{t('settings.advanced.clearCache.help8', 'Installed games')}</li>
+          <li>{t('settings.advanced.clearCache.help9', 'Games settings')}</li>
+          <li>
+            {t('settings.advanced.clearCache.help10', 'Heroic configuration')}
+          </li>
+        </ul>
```

Comment:
I'd be great if it would be possible to make this whole list freely translatable (I worry about languages not conforming to this format), but I don't think i18next offers something for this

Reply:
yeah, I don't really know how to mix a list with translations as a single translation (I could add `{{}}` placeholder to replace with the html tags, but it would be a mess as a single string)

Reply:
I don't think that'd work actually (it would just display the tags as text, e.g. `<ul><li>foo</li>...`). You'd have to use `dangerouslySetInnerHTML`, which is, well, dangerous